agent/exploits/cve_2024_43044.py

```diff
@@ -0,0 +1,81 @@
+"""Agent Asteroid implementation for CVE-2024-43044"""
+
+import re
+from packaging import version
+from requests import exceptions as requests_exceptions
+import requests
+from urllib import parse as urlparse
+
+from agent.exploits import webexploit
+from agent import exploits_registry
+from agent import definitions
+
+
+VULNERABILITY_TITLE = "JENKINS ARBITRARY FILE READ TO RCE"
+VULNERABILITY_REFERENCE = "CVE-2024-43044"
+VULNERABILITY_DESCRIPTION = """Arbitrary file read vulnerability through agent connections can lead to RCE 
```

Comment:
```suggestion
VULNERABILITY_DESCRIPTION = """Arbitrary file read vulnerability through agent connections can lead to RCE.
```
Also, I think you need a bit more details..